MakshakovDD/lab_1/main.cpp

```diff
@@ -0,0 +1,61 @@
+﻿	#include <iostream>
+	#include <string>
+	#include <algorithm>
+	#include <vector>
+	#include <cctype>
+	using namespace std;
+
+	struct Punctuation {
+		char symbol;
+		int words_before;
+	};
+
+	bool compare(string a, string b){
```

Comment:
Current comparator implementation is not very efficient in terms of performance
std::sort will perform N*log(N) comparisons and for each of them you will count sum of ascii characters 
Please rework it in a way when sum for each word will be computed only once

Reply:
Also consider passing `a` and `b` by const reference